test/unit/Editing.test.ts

```diff
@@ -43,4 +43,11 @@ describe('Editing Element', () => {
     expect(element).to.have.attribute('name', 'A2');
     expect(element).to.not.have.attribute('desc');
   });
+
+  it('removes an element on Remove Action', () => {
+    editor.dispatchEvent(newOpenDocEvent(doc, 'test.scd'));
+    const node = doc.querySelector('Substation')!;
+    editor.dispatchEvent(newActionEvent({ node }));
+    expect(doc.querySelector('Substation')).to.not.exist;
```

Comment:
This test is also not fail save. Someone could rewrite that the function does remove the whole XMLDocument and we would not see from the test. How about testing that the parent still exists. And maybe that only the one node from the parent has been removed?

Reply:
Maybe we can use https://www.npmjs.com/package/@open-wc/semantic-dom-diff to compare expected dom to actual dom?

Reply:
Also, let's not go for "fail-safe" tests documenting current behaviour after the fact, but instead use tests to specify behaviour piece-meal. This other type of test I would rather see as a regression test for an actually introduced bug.

I've been trying to develop this in an actually test driven way, something along these lines: https://www.youtube.com/watch?v=EZ05e7EMOLM